packages/deployments/contracts/src/cli/init/config/testnet/production.ts

```diff
@@ -132,6 +132,21 @@ export const TESTNET_PRODUCTION_INIT_CONFIG: InitConfig = {
         },
       },
     },
+    {
+      name: "xDappRadar",
+      canonical: {
+        domain: "1735353714",
+        address: "0x369e5C58F6c09c1CE947CA313f03c0d558B77b3e",
+        decimals: 18,
+      },
+      representations: {
+        /// OPTIMISM-GOERLI
+        "1735356532": {
+          local: "0xb6932c8363354df7620a7a7bF3a933A253015cA3",
+          adopted: "0xb6932c8363354df7620a7a7bF3a933A253015cA3",
+        },
```

Comment:
Can we add token for mumbai into representations part
` 
"9991": {
      local: "0x4c82266B29F82525FD20560638C517D11219F5B5",
      adopted: "0x4c82266B29F82525FD20560638C517D11219F5B5",
    }
`

Reply:
Yeah ofcouse. You can